app/templates/components/forms/profile-page.hbs

```diff
@@ -0,0 +1,34 @@
+<form class="ui large form" autocomplete="off" {{action 'submit' on='submit'}}>
+  <div class="ui stackable grid">
+    <div class="four wide column">
+      {{widgets/forms/image-upload
+        imageUrl=data.event.backgroundUrl
+        needsCropper=true
+        label=(t 'Event Image')
+        id='event_image'
+        icon='camera'
+        hint=(t 'Select Event Image')
+        maxSizeInKb=10000
+        helpText=(t 'We recommend using at least a 2160x1080px (2:1 ratio) image')}}
+    </div>
+    <div class="ten wide column">
+      <div class="field">
+        <label class="required" for="name">{{t 'Name'}}</label>
+        {{input type='text' id='name'}}
+      </div>
+      <div class="field">
+        <label class="required" for="family-name">{{t 'Family Name'}}</label>
+        {{input type='text' id='family-name'}}
+      </div>
+      <div class="field">
+        <label for="description">{{t 'Details about user'}}</label>
+        {{widgets/forms/rich-text-editor id='description'}}
+      </div>
+      <div class="ui center aligned segment basic">
```

Comment:
@niranjan94 Currently it's a fluid button, you want to it to remain fluid to take the entire width of the container, or a small button aligned on the left side?